llvm/test/DebugInfo/Symbolize/XCOFF/xcoff.test

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
Nit: please add comments explaining the purpose of this test.

This test is called "xcoff.test", implying it's supposed to be about generic xcoff functionality, but the patch is specifically about symbol sizes. Can the test be more focused?

Reply:
Thanks, the test file name is changed to be more focused. This test is from https://github.com/llvm/llvm-project/pull/67284. The bug was detected there. I added a `FIXME` in the base test and in this PR, the `FIXME` is removed and correct size is shown. See the final commit https://github.com/llvm/llvm-project/pull/67304/commits/b9a3a7a6cda9d98520a7d314b4cbdd839407fce6 after PR https://github.com/llvm/llvm-project/pull/67284 is merged.